tests/unit/retroGenInit.unitTest.ts

```diff
@@ -1,10 +1,8 @@
 import { SQService } from "../../src/services/SQService";
```

Comment:
Not sure if anyone else is having the same issue, but I am having test failures when running locally after dep reinstall and project rebuild.